src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -76,14 +77,17 @@ public Task GetClusterTopology()
                 
                 using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
                 {
-                    context.Write(writer, new DynamicJsonValue
+                    var json = new DynamicJsonValue
                     {
                         ["Topology"] = blit,
-                        ["Errors"] = ServerStore.GetClusterErrors(),
                         ["Leader"] = ServerStore.LeaderTag,
                         ["CurrentState"] = ServerStore.CurrentState,
                         ["NodeTag"] = nodeTag
-                    });
+                    };
+                    if (ServerStore.GetClusterErrors().Count > 0)
```

Comment:
I suggest assigning result from GetClusterErrors to variable to avoid double execution.